src/pipx/main.py

```diff
@@ -183,6 +183,18 @@ def package_is_url(package: str, raise_error: bool = True) -> bool:
     return False
 
 
+def package_is_abspath(package: str):
+    if Path(package).is_absolute() or Path(package).exists():
```

Comment:
This will fail when running commands in the directory that contains the local source folder:
```
~ $ ls
black
~ $ pipx upgrade black
Error: Path 'black' given as package. Expected the name of an installed package.
```

I suggest that we change the `Path(package).exists()` check:
```suggestion
    if Path(package).is_absolute() or or "/" in package:
```

Reply:
Or we can simply have a check like this:
```suggestion
    if "\\" or "/" in package:
```

Reply:
I'm fine with the suggestion, but I'd rewrite it to use `os.path.sep`.